ops/services/app_functions/report_stream_batched_publisher/functions/common/queueHandlers.test.ts

```diff
@@ -0,0 +1,447 @@
+import {
+  deleteSuccessfullyParsedMessages,
+  dequeueMessages,
+  getQueueClient,
+  minimumMessagesAvailable,
+  reportExceptions,
+} from "./queueHandlers";
+import {
+  DequeuedMessageItem,
+  QueueClient,
+  QueueDeleteMessageResponse,
+  QueueServiceClient,
+  StorageSharedKeyCredential,
+} from "@azure/storage-queue";
+import { ReportStreamError, ReportStreamResponse } from "./rs-response";
+import { Context } from "@azure/functions";
+
+jest.mock("../config", () => ({
+  ENV: {
+    AZ_STORAGE_QUEUE_SVC_URL: "hello",
+    AZ_STORAGE_ACCOUNT_NAME: "hola",
+    AZ_STORAGE_ACCOUNT_KEY: "bonjour",
+    TEST_EVENT_QUEUE_NAME: "ciao",
+    REPORT_STREAM_URL: "https://nope.url/1234",
+    REPORT_STREAM_TOKEN: "merhaba",
+    REPORT_STREAM_BATCH_MINIMUM: "1",
+    REPORT_STREAM_BATCH_MAXIMUM: "5000",
+  },
+}));
+
+jest.mock("@azure/storage-queue");
+
+const queueServiceClientMock = QueueServiceClient as jest.MockedClass<
```

Comment:
Github diffs are sadly no good at showing moved code - is it safe to assume all of this file was copied over from lib.test.ts?

Reply:
yes, that is correct!. All the methods from lib.ts that were related to handling the queue were moved to the queueHandlers.ts file and all the unit tests from lib.test.ts that covered those methods was migrated to queueHandlers.test.ts. No coverage was added or removed in the migration :D.